templates/tfengine/components/cicd/main.tf

```diff
@@ -134,6 +134,22 @@ resource "google_project_iam_member" "cloudbuild_builds_editors" {
     google_project_service.services,
   ]
 }
+
+# IAM permission to allow approvers to trigger builds.
+resource "google_service_account_iam_member" "cloudbuild_builds_editors" {
+  for_each = toset([
+    {{- range .build_editors}}
+    "{{.}}",
+    {{- end}}
+  ])
+  service_account_id = google_service_account.cloudbuild_sa.name
```

Comment:
Should use `local.cloudbuild_sa_email` here instead.

Reply:
Fixed